src/agent/commands/queries.js

```diff
@@ -17,6 +17,13 @@ export const queryList = [
             let pos = bot.entity.position;
             // display position to 2 decimal places
             res += `\n- Position: x: ${pos.x.toFixed(2)}, y: ${pos.y.toFixed(2)}, z: ${pos.z.toFixed(2)}`;
+            // Environmental Awareness
+            res += `\n- Block Above: ${world.getBlockAtPosition(bot, 0, 2, 0).name}`;
+            res += `\n- Block Below: ${world.getBlockAtPosition(bot, 0, -1, 0).name}`;
+            res += `\n- Block at Head: ${world.getBlockAtPosition(bot, 0, 1, 0).name}`;
+            res += `\n- Block at Legs: ${world.getBlockAtPosition(bot, 0, 0, 0).name}`;
+            res += `\n- Lowest Block Above: ${world.getLowestBlock(bot, null, null, 32).name}`;
```

Comment:
can you rename "Lowest Block Above" to "First Solid Block Above Head". I think that is clearer.

Reply:
oh also you should include how high above the head it is, where 0 is right above. 
First Solid Block Above Head: stone (23 blocks up)